### ISSUE TITLE
what the mean about the `\main.loader.js`

### ISSUE THREAD

**Issue by godtoy** [herberttn/bytenode-webpack-plugin#4]
![image](https://user-images.githubusercontent.com/13638996/116535586-6e7a3280-a8d3-11eb-91f7-2286a87939c0.png)

```
 error  in ./src/renderer/main.loader.js

Module build failed (from ./node_modules/thread-loader/dist/cjs.js):
Thread Loader (Worker 0)
```
yo guys, im using vue-cli-electron-builder lib, when i add your plugin to build it mind me module cannot be found

**Comment by godtoy** [comment]
the main process is worked, but not work on rederder process

**Comment by godtoy** [comment]
![image](https://user-images.githubusercontent.com/13638996/116542261-ffeda280-a8db-11eb-8deb-9a4b7b431f6c.png)
it's my setting

**Comment by godtoy** [comment]
and i think when im building renderder module, virtualModules not work

**Comment by godtoy** [comment]
stopped here 
![Uploading image.png…]()

**Comment by godtoy** [comment]
another question is if i want to ignore some files, what should i do

**Comment by godtoy** [comment]
i created same issue at here, https://github.com/nklayman/vue-cli-plugin-electron-builder/issues/1346

**Comment by godtoy** [comment]
Also sometimes i want to ignore some script dont create jsc `https://github.com/sbrow/bytenode-webpack-plugin` this lib have done this feature

**Comment by godtoy** [comment]
i think that's maybe `https://github.com/vuejs/vue-cli` this project issue, and it's working on vue-electreon framework

**Comment by herberttn** [comment]
I don't know how `vue-cli-electron-builder` works, but `webpack-virtual-modules` should work regardless anyway. I think yours is a different problem. Do you have a repro repo?

I'm planning to add `include` and `exclude` options soon.

**Comment by godtoy** [comment]
yep, i have , pls see this repo `https://github.com/godtoy/vue-cli-electron-bytenode-issue-6451`

**Comment by godtoy** [comment]
thanks y

> I don't know how `vue-cli-electron-builder` works, but `webpack-virtual-modules` should work anyway.
> I think yours is a different problem. Do you have a repro repo?
> 
> I'm planning to add `include` and `exclude` options soon.

thanks man

**Comment by godtoy** [comment]
Yo, man. anything new ?

**Comment by herberttn** [comment]
Hello @godtoy!
I'll probably try to test this in the next few days...
I was really busy finishing another project.

**Comment by godtoy** [comment]
> 你好@godtoy！
> 我可能会在接下来的几天中进行测试...
> 我真的很忙于完成另一个项目。

thank you so much

**Comment by godtoy** [comment]
hello, any update ?

**Comment by jiumengs** [comment]
@herberttn May I ask if you have fixed this problem? I also encountered the same problem   src/main.loader.js module could not be found

**Comment by godtoy** [comment]
no yet

jiumeng ***@***.***> 于2021年8月24日周二 下午7:09写道：

**Comment by zhaoruilinda** [comment]
@herberttn May I ask when you plan to fix this problem? I also encountered the same problem src/main.loader.js module could not be found

**Comment by godtoy** [comment]
give up to encrypt the renderer process codes. it only works on main.js
![image](https://user-images.githubusercontent.com/13638996/153557149-799ea6fe-e34d-4c6d-bf9d-add5b8d35693.png)

**Comment by herberttn** [comment]
I've never tried to use this plugin with `vue-cli-electron-builder` or `electron-builder`, so I don't know if it works or what your error could be.

From my point of view, this is a feature request and not exactly a bug to be fixed. So, since this is not a bug and I don't use these tools, I haven't had the time to check this out yet.

Please feel free to open a pull request adding support for these tools or solving the problem you encountered, that would be nice!

**Comment by devzw** [comment]
@herberttn May I ask if you have fixed this problem? I also encountered the same problem src/main.loader.js module could not be found

**Comment by absir1949** [comment]
i have the same problem.
